### PR TITLE
Interactions

### PR DESCRIPTION
**Pull request by Gmedulla101**
Changed the responsive navbar from a drop down menu to a modal. Added the necessary JS functionality to it and fixed the issue of the navbar not sticking to the header when the page is scrolled.

### CODE REVIEW

**Review by Gmedulla101 — COMMENTED** [review]
well written code and it works

File: main.css [hunk 1]
```diff
@@ -907,6 +910,30 @@ footer .footer-container p{
...
+    }
+    #modal {
+        display: none;
+        align-items: center;
+        justify-content: center;
+        position: absolute;
+        background: rgba(40, 58, 90, 0.9);
+        width: 100%;
+        height: 43em;
+        z-index: 1000000000000;
```
Comment (UBEPSAweb) [inline main.css:925]: No need for the astronomical z-index

File: main.js [hunk 2]
```diff
@@ -1,16 +1,18 @@
...
+const cancel = document.querySelector('#closeBtn');
+const modal = document.querySelector('#modal');
+
+//ADDING AN EVENT LISTENER TO THE BURGER BUTTON TO CALL THE MODAL
+burger.addEventListener("click", (event) => {
+  modal.style.display = "flex";
+});
+
+//ADDING AN EVENT LISTENER TO CLOSE THE MODAL
+cancel.addEventListener('click', (event) => {
```
Comment (UBEPSAweb) [inline main.js:13]: You could make this a function and add it to each menu option so that the modal closes when they are clicked

**Review by UBEPSAweb — APPROVED** [review]
Overall good code and well written code. Easily readable.

**Pull request merged**